de-lib/src/main/java/org/iplantc/de/theme/base/client/admin/toolRequest/ToolRequestViewDefaultAppearance.java

```diff
@@ -14,19 +20,27 @@
     private final ToolRequestDisplayStrings displayStrings;
     private final IplantResources iplantResources;
     private final IplantDisplayStrings iplantDisplayStrings;
+    private final Templates templates;
+
+
+    interface Templates extends SafeHtmlTemplates {
+        @SafeHtmlTemplates.Template("<img qtip='{1}' src='{0}'/> <span>{1}</span> ")
+        SafeHtml makePublicRequest(SafeUri img, String name);
+    }
 
     public ToolRequestViewDefaultAppearance() {
         this(GWT.<ToolRequestDisplayStrings> create(ToolRequestDisplayStrings.class),
              GWT.<IplantDisplayStrings> create(IplantDisplayStrings.class),
-             GWT.<IplantResources> create(IplantResources.class));
+             GWT.<IplantResources> create(IplantResources.class), GWT.<Templates>create(Templates.class));
     }
 
     ToolRequestViewDefaultAppearance(final ToolRequestDisplayStrings displayStrings,
                                      final IplantDisplayStrings iplantDisplayStrings,
-                                     final IplantResources iplantResources) {
+                                     final IplantResources iplantResources, final Templates templates ) {
```

Comment:
Minor formatting issue: the new `Templates` param in these constructors should be on a new line.